app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    orig_file, new_file = command[3:].split(" ")
```

Comment:
u could unpack it like this
var1,var2,var3 = command.split()
to avoid using indexes